### DISCUSSION TITLE
[Bug] [monaco-typescript] completion items are not showing for overload function parameters

### DISCUSSION THREAD

**Discussion by ttys026** [microsoft/monaco-editor#3709] [Q&A]
### Reproducible in vscode.dev or in VS Code Desktop?

- [X] Not reproducible in [vscode.dev](https://vscode.dev) or VS Code Desktop

### Reproducible in the monaco editor playground?

- [ ] Not reproducible in [the monaco editor playground](https://microsoft.github.io/monaco-editor/playground.html)

### Monaco Editor Playground Code

```typescript
monaco.editor.create(document.getElementById('container'), {
	value: `interface Hello {
    (name: 'a'): void;
    (name: 'b'): void;
    (name: 'c'): void;
    (name: 'd'): void;
}
const hello: Hello = name => {};

hello()
  `,
	language: 'typescript'
});
```

### Actual Behavior

when start to type the single quote as the parameter of `hello` function, there is no IntelliSense showing up.

![image](https://user-images.githubusercontent.com/19804057/172681315-0a06878f-71a5-4ccc-bb05-2bcda999ec2b.png)

### Expected Behavior

when start to type the single quote as the parameter of `hello` function, there is a IntelliSense showing up.

![image](https://user-images.githubusercontent.com/19804057/172681257-1dcfd772-40c1-4df0-8adc-b96e247771da.png)

**Comment by hediet** [comment]
IntelliSense works:

![image](https://user-images.githubusercontent.com/2931520/179767391-21598947-d326-4d85-965b-cc1a6e916655.png)

I think the issue here are trigger characters and/or quick suggest.
